README.md

```diff
@@ -13,9 +13,13 @@ On a Debian-based system
 
 ## How to use on your SBC ##
 
-    echo deb [trusted=yes] https://github.armbian.com/apa current main | sudo tee /etc/apt/sources.list.d/armbian-apa.list
-    sudo apt update
```

Comment:
dropping this and the following line is certainly wrong

Reply:
I am still unable to reproduce the issue that this PR is trying to fix.  Requesting again for more details on how to reproduce whatever issue you are seeing.

Reply:
In other words, your second commit is addressing an issue I didn't have but completely skips over all the issues I raised here.

Reply:
I lost you, sorry. Please be specific or fix the commit what you think its wrong here. Point of this PR is changing .list to .sources.